app/models/user.rb

```diff
@@ -1,4 +1,12 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] _Confirmable_ is disabled. As we want the user(s) to be able to confirm their credentials kindly ensure to enable _confirmable_.